pyproject.toml

```diff
@@ -0,0 +1,15 @@
+[tool.poetry]
+name = "microdata-crypt"
+version = "0.1.0"
+description = ""
+authors = ["Kalantzakos, Vassilios <Vassilios.Kalantzakos@ssb.no>"]
```

Comment:
Be mindful that your personal email is in this public repo. (this might be intended so the users can send a mail easily) I just thought I should point it out 👍🏻